restfull-api/routes/todos.js

```diff
@@ -1,4 +1,72 @@
 const express = require("express");
 const router = express.Router();
+const Todo = require("../models/todo");
+const todo = require("../models/todo");
+
+//getting all
+router.get("/", async (req, res) => {
+  try {
+    const todos = await Todo.find();
+    res.json(todos);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+//getting one
+router.get("/:id", getTodo, (req, res) => {
+  res.json(res.todo);
+});
+
+//creating one
+router.post("/", async (req, res) => {
+  const todo = new Todo({
+    todoText: req.body.todoText,
```

Comment:
Use the library `Joi` to validate the request payload before processing it

Reply:
abi done